data/en/getsoaprequest.json

```diff
@@ -1,16 +1,27 @@
 {
 	"name":"getSOAPRequest",
 	"type":"function",
-	"syntax":"getSOAPRequest()",
+	"syntax":"getSOAPRequest(webservice)",
 	"returns":"any",
 	"related":["getSOAPRequestHeader","getSOAPResponse","getSOAPResponseHeader"],
 	"description":" Returns an XML object that contains the entire SOAP\n request. Usually called from within a web service CFC.",
-	"params": [],
+	"params": [
+		{"name":"webservice","description":"A webservice object as returned from the cfobject tag or the CreateObject function.","required":true,"default":"","type":"any","values":[]}
```

Comment:
I don't think this argument `required` according to adobe docs it is optional

Reply:
I can update, I see that Lucee and the Adobe docs have that.  Also I tried to confirm that by using trycf.com and the following code fails on Lucee and ColdFusion 2018 and ColdFusion 2016:
```
soapURL = "https://graphical.weather.gov/xml/SOAP_server/ndfdXMLserver.php?wsdl";
ws = CreateObject("webservice", soapURL);
zipLatLong = ws.LatLonListZipCode("10001");
//res = getSOAPRequest(ws);
res = getSOAPRequest();
writeDump(res);
```